application/src/main/resources/application-default.yml

```diff
@@ -1,3 +1,22 @@
 hono:
   registration:
     filename: /home/hono/registration/device-identities.json
+  metric:
+    jvm:
+      memory: true
+      thread: true
+    reporter:
+      console:
+        active: false
+        period: 5000
+      graphite:
```

Comment:
If we do not include the Graphite reporter in the inital PR then we also do not need to set properties for it, right?